tests/puppeteer/campaigns/data/faker/customer.js

```diff
@@ -0,0 +1,22 @@
+const faker = require('faker');
+
+const genders = ['Mr.', 'Mrs.'];
+const defaultCustomerGroups = ['Visitor', 'Guest', 'Customer'];
+
+module.exports = class Customer {
+  constructor(customerToCreate = {}) {
+    this.socialTitle = customerToCreate.socialTitle || faker.random.arrayElement(genders);
+    this.firstName = customerToCreate.firstName || faker.name.firstName();
+    this.lastName = customerToCreate.lastName || faker.name.lastName();
+    this.email = customerToCreate.email || `${this.firstName}.${this.lastName}@prestashop.com`;
```

Comment:
No problem with special names, for example with a space or a dash ?

Reply:
I didn't have a problem in all my tests + tests on [faker](https://rawgit.com/Marak/faker.js/master/examples/browser/index.html#name), but i can't found source to confirm that.
Do you think using `faker.internet.exampleEmail` should be better ?

Reply:
Changed with
```js
    this.email = customerToCreate.email || faker.internet.email(this.firstName, this.lastName, 'prestashop.com');
```